image_classification/README.en.md

```diff
@@ -395,106 +313,131 @@ The following are the components of `resnet_cifar10`:
 ```
 
-- `--config=$cfg` : specifies config file. The default is `models/vgg.py`.
-- `--use_gpu=true` : uses GPU for training. If use CPU，set it to be false.
-- `--trainer_count=1` : specifies the number of threads or GPUs.
-- `--log_period=100` : specifies the number of batches between two logs.
-- `--save_dir=$output` : specifies the path for saving trained models.
+### Construct Trainer
 
-Here is an example log after training for one pass. The average error rates are 0.79958 on training set and 0.7858 on validation set.
+Before jumping into creating a training module, algorithm setting is also necessary.
+Here we specified `Momentum` optimization algorithm via `paddle.optimizer`.
 
-```text
-TrainerInternal.cpp:165]  Batch=300 samples=38400 AvgCost=2.07708 CurrentCost=1.96158 Eval: classification_error_evaluator=0.81151  CurrentEval: classification_error_evaluator=0.789297
-TrainerInternal.cpp:181]  Pass=0 Batch=391 samples=50000 AvgCost=2.03348 Eval: classification_error_evaluator=0.79958
-Tester.cpp:115]  Test samples=10000 cost=1.99246 Eval: classification_error_evaluator=0.7858
+```python
+# Create optimizer
+momentum_optimizer = paddle.optimizer.Momentum(
+    momentum=0.9,
+    regularization=paddle.optimizer.L2Regularization(rate=0.0002 * 128),
+    learning_rate=0.1 / 128.0,
+    learning_rate_decay_a=0.1,
+    learning_rate_decay_b=50000 * 100,
+    learning_rate_schedule='discexp',
+    batch_size=128)
+
+# Create trainer
+trainer = paddle.trainer.SGD(cost=cost,
+                             parameters=parameters,
+                             update_equation=momentum_optimizer)
 ```
 
-Figure 12 shows the curve of training error rate, which indicates it converges at Pass 200 with error rate 8.54%.
+The learning rate adjustment policy can be defined with variables `learning_rate_decay_a`($a$), `learning_rate_decay_b`($b$) and `learning_rate_schedule`. In this example, discrete exponential method is used for adjusting learning rate. The formula is as follows,
+$$  lr = lr_{0} * a^ {\lfloor \frac{n}{ b}\rfloor} $$
+where $n$ is the number of processed samples, $lr_{0}$ is the learning_rate.
 
-<p align="center">
-<img src="image/plot_en.png" width="400" ><br/>
-Figure 12. The error rate of VGG model on CIFAR10
-</p>
+### Training
 
-## Model Application
+`cifar.train10()` will yield records during each pass, after shuffling, a batch input is generated for training.
 
-After training is done, the model from each pass is saved in `output/pass-%05d`. For example, the model of Pass 300 is saved in `output/pass-00299`. The script `classify.py` can be used to extract features and to classify an image. The default config file of this script is `models/vgg.py`.
+```python
+reader=paddle.reader.batch(
+    paddle.reader.shuffle(
+        paddle.dataset.cifar.train10(), buf_size=50000),
+        batch_size=128)
+```
 
+`feeding` is devoted to specify the correspondence between each yield record and `paddle.layer.data`. For instance,
```

Comment:
is devoted to specify -> is devoted to specifying （devoted to something:)）
Or:
is used to specify